expertsystem/io/_pdg.py

```diff
@@ -0,0 +1,171 @@
+"""Create a `.ParticleCollection` instance from PDG info."""
```

Comment:
Why is this module private?

Reply:
To hide implementation details. I want to keep the `io.__init__.py` clean because it's accumulating a lot of functionality (probably even #216).

Reply:
But everything is already private in there except `load_pdg()`. So basically you want to be able to do `io.load_pdg()` instead of `io.pdg.load_pdg()`? And what is the point of hiding the implementation detail of a public function?

Reply:
It's about how the Sphinx API renders

Reply:
https://pwa--190.org.readthedocs.build/projects/expertsystem/en/190/api/expertsystem.io.html#module-expertsystem.io

Reply:
I see. But why not move the `load_pdg()` function from `_pdg.py` to `__init__.py`?

Reply:
Have a look e962dcd785d68b566ebe1f1e6245f347dda0dda7

Reply:
Actually I prefer the previous way :grimacing:

Reply:
No problem :sweat_smile: 325f62bf801c87bf81615bd534fa94cdd5de278e